src/core.js

```diff
@@ -1,3 +1,4 @@
+d3 = require('d3');
```

Comment:
I assume they're actually tryign to use requirejs there?   There is another pull request for AMD, we should probably just make it loadable for requriejs and AMD so people don't have to use a shim.